src/elements/common/messages.js

```diff
@@ -1015,6 +1015,11 @@ const messages = defineMessages({
         defaultMessage: 'Box Doc Gen',
         description: 'Icon title for a Box file of type DocGen template',
     },
+    loading: {
+        id: 'be.loading',
+        description: 'Loading aria label.',
+        defaultMessage: 'loading',
```

Comment:
does this need to be capitalized? i don't think so since it's just an aria label right?

Reply:
I dont think they annunciate it any differently with the screen readers